src/librbd/Utils.cc

```diff
@@ -31,5 +39,38 @@ librados::AioCompletion *create_rados_ack_callback(Context *on_finish) {
   return create_rados_ack_callback<Context, &Context::complete>(on_finish);
 }
 
+int parse_rbd_default_features(CephContext* cct, const std::string& str_value, uint64_t* value) 
```

Comment:
As we always ignore return value I think it is better to return features. Also, it seems we always use cct->_conf->rbd_default_features as str_value, so may be do this inside this function, and rename the function just to rbd_default_features()?

It would simplify the code to:
```
uint64_t features = old_format ? 0 : utils::rbd_default_features(cct);
```
instead of
```
uint64_t features;
      if (old_format)
        features = 0;
      else {
        librbd::util::parse_rbd_default_features(cct, cct->_conf->rbd_default_features, &features);
      }
```